src/pages/index.js

```diff
@@ -174,7 +174,9 @@ function Home() {
                   "button button--outline button--secondary button--lg button--hero get-started-use-case",
                   styles.getStarted
                 )}
-                to={useBaseUrl("docs/guides/orchestrate-microservices/")}
+                to={useBaseUrl(
+                  "docs/guides/getting-started-orchestrate-microservices/"
```

Comment:
This page isn't versioned, so these URLs were pointing at 8.2 pages. I modified them to point at similar 1.3 pages.